cmd/src/debug_comp.go

```diff
@@ -0,0 +1,255 @@
+package main
+
+import (
+	"archive/zip"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"golang.org/x/sync/errgroup"
+
+	"golang.org/x/sync/semaphore"
+
+	"github.com/sourcegraph/src-cli/internal/cmderrors"
+)
+
+func init() {
+	usage := `
+'src debug comp' mocks docker cli commands to gather information about a docker-compose Sourcegraph instance.
+
+Usage:
+
+    src debug comp [command options]
+
+Flags:
+
+	-o			Specify the name of the output zip archive.
+	-cfg		Include Sourcegraph configuration json. Defaults to true.
```

Comment:
This formatting seems odd

Reply:
You're right 😢

Reply:
improved across functions to use flag `-no-config` which is false by default